src/Resources/UserResource.php

```diff
@@ -29,6 +29,12 @@ class UserResource extends Resource
 
     protected static Closure | null $extendFormCallback = null;
 
+    protected static ?string $navigationLabel = null;
```

Comment:
We don't need these properties to be defined on the child class, the `Resource` parent class already has them.

Reply:
Yes, we already have them and it makes sense to avoid defining them again. But if we don't define it, it will overwrite the properties of the `Resource` main class and all resources will have the same label by default.

I did the experiment:
- Declaring the properties: https://d.pr/i/TlAO2R
- Without declaring the properties: https://d.pr/i/7Y1Goe

To reproduce this, you have to set the labels on your `AppServiceProvider`:
```php
UserResource::label(__('Utilizador'));
UserResource::pluralLabel(__('Utilizadores'));
```